src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="block block1">1</div>
+    <div class="block block2">2</div>
+    <div class="block block3">3</div>
+    <div class="block block4">4</div>
+    <div class="block block5">5</div>
+    <div class="block block6">6</div>
```

Comment:
we'll learn that later, in BEM topic, but what you have here (block1, block2 etc.) are a modifiers. They slightly change the state of the element. Syntactically they should look like that:
```suggestion
    <div class="block block--1">1</div>
    <div class="block block--2">2</div>
    <div class="block block--3">3</div>
    <div class="block block--4">4</div>
    <div class="block block--5">5</div>
    <div class="block block--6">6</div>
```